graphql/errors.go

```diff
@@ -41,6 +50,10 @@ func NewSafeError(format string, a ...interface{}) error {
 	return SafeError{message: fmt.Sprintf(format, a...)}
 }
 
+func WrapAsSafeError(err error, format string, a ...interface{}) error {
+	return SafeError{inner: err, message: fmt.Sprintf(format, a...)}
```

Comment:
how is this message going to be used? if our impl is going to use a middleware to wrap other errors that already have messages, i think we don't need it?

Reply:
We want to extract the message from the error, and use that. Thunder will still use `sanitizeError()` for the message, and thus SafeError will still need to provide that

Reply:
can we use the underlying message from the err as `inner` instead of asking the caller to provide one?

Reply:
Talked about this in person.

To have thunder extract messages itself, would require it to know and understand our error patterns and structure. I'd rather defer the logic of how what goes in the message and how we unwrap it to added middleware